### PR TITLE
Feature/shreyagupta112/created user model integrted passport js and made admin get route

### PR DESCRIPTION
**Pull request by shreyagupta112**
### Administrative Info
#20 

### Changes
What changes did you make?
Created the model.js, auth.js, scure-routes.js, routes.js, and admin.js files. Model.js is the user schema, admin.js contains a get route which prints out a list of admin or non-admin users, and the other three files deal with authentication.

Testing

### Testing
How did you confirm your changes worked?
I created a few fake users to test out the admin get route and authentication and got it to work locally.

### Confirmation of Change 
Upload a screenshot, if possible. Otherwise, please provide instructions on how to see the change.
I created these files, and haven't submitted any pull requests yet, so the entirety of the files are a new change.

### CODE REVIEW

**Review by PatrickBrown1 — CHANGES_REQUESTED** [review]
I made a lot of changes to the schemas and routes, so now all of the authorization routes are functional. 

The only route left to do is the get users routes.
Can you change the route name to be /getUsers, and take one query param called admin. This query param should be verified as a boolean. If it is true, then it will return all users who are admins. If it is false, it will return all users who are not admins.

File: backend/models/UserModel.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        index: true},
+    validated: {type: Boolean, default: false}
+    
+
+}); */
+
+const UserSchema = new Schema({
+  verified: {
+    type: Boolean,
+    //  required: true,
```
Comment (mohakvni) [inline backend/models/UserModel.js:26]: I think its better to make verified required

**Review by mohakvni — COMMENTED** [review]
Looks good to me, apart from the changes Patrick requested, I have added a comment in UserModel for making verified required.

**Comment by PatrickBrown1** [comment]
LGTM, will approve when @mohakvni gets a chance to take a look

**Review by mohakvni — APPROVED** [review]
LGTM!

**Review by PatrickBrown1 — APPROVED** [review]

**Pull request merged**